modules/web_framework.rb

```diff
@@ -0,0 +1,107 @@
+
+require 'selenium-webdriver'
+
+class Web_framework
+  @url = nil
+  def initialize()
+    Selenium::WebDriver::Chrome::Service.driver_path=ENV['driver']
+    options = Selenium::WebDriver::Chrome::Options.new
+        options.add_argument('--headless')
+        options.add_argument('--no-sandbox')
+        options.add_argument('--disable-dev-shm-usage')
+        options.add_argument('--window-size=1920,1080')
+        @driver = Selenium::WebDriver.for :chrome ,options: options
+  end
+  def get_webpage(url)
+    @url = url
+    @driver.get(url)
+  end
+  def click_element(element)
+    begin
+      return element.click
+    rescue
+      return "null element"
```

Comment:
By returning `"null element"`, it is not possible to ascertain what the actual error is